navigation/libandroid-navigation-ui/src/main/res/layout/navigation_view_layout.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/navigationLayout"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.mapbox.mapboxsdk.maps.MapView
+        android:id="@+id/mapView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:mapbox_cameraTargetLat="40.246"
```

Comment:
If you are immediately moving the map to the passed in latlng values inside the bundle, you shouldn't set the map camera in XML and cause unnecissary resources to be loaded in.

Reply:
Removed unneeded lat lng and zoom level